src/ar_tx_search.erl

```diff
@@ -146,8 +146,17 @@ ensure_table_exists() ->
 		{atomic, ok} -> ok;
 		{aborted, {already_exists, arql_tag}} -> ok
 	end,
-	ok = mnesia:wait_for_tables([arql_tag], 5000),
-	ok.
+	case mnesia:wait_for_tables([arql_tag], 5000) of
```

Comment:
Why should we call this function. I think we should just replace it with `force_table_load`. Otherwise we just introduce useless overhead